data/presets_default.json

```diff
@@ -1325,40 +1354,37 @@
         "fast_bunny_hood": true,
         "plant_beans": false,
         "chicken_count_random": false,
-        "chicken_count": 7,
+        "chicken_count": 3,
         "big_poe_count_random": false,
         "big_poe_count": 1,
         "easier_fire_arrow_entry": false,
+        "fae_torch_count": 3,
         "ocarina_songs": "off",
-        "correct_chest_appearances": "classic",
+        "correct_chest_appearances": "both",
         "minor_items_as_major_chest": false,
         "invisible_chests": false,
         "correct_potcrate_appearances": "textures_content",
         "clearer_hints": true,
         "hints": "always",
-        "hint_dist": "league",
+        "hint_dist": "leagueS4",
```

Comment:
Remember to adjust this when fixing the hint distro naming inconsistency.

Reply:
Changed as part of the other conversation, now back to just "league"